includes/connect/class-wc-stripe-connect.php

```diff
@@ -154,6 +156,9 @@ private function save_stripe_keys( $result, $mode = 'live' ) {
 
 			update_option( self::SETTINGS_OPTION, $options );
 
+			// Automatically configure webhooks for the account now that we have the keys.
+			WC_Stripe::get_instance()->account->configure_webhooks( $is_test ? 'test' : 'live', $secret_key );
```

Comment:
The exception thrown in the `configure_webhooks` is not handled here. For testing purpose, I intentionally created an error condition [in the function](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3293/files#diff-4314dc04e3c11401b896736a579e7b2cce4634bab67b790ef2872cc6b39edfb3R300) and got this fatal error.
 
<img width="815" alt="Screenshot 2024-08-02 at 5 43 40 PM" src="https://github.com/user-attachments/assets/501de6a6-7e2c-4b9c-922f-99f018239098">

Reply:
Good call. I've fixed this in [e74c110](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3293/commits/e74c110d1c31b8c975f64ef9194f4b28e6b24add) by catching an exception and returning a `WP_Error` which is consistant with this function returning error objects.